app/routes/_auth+/auth.$provider.ts

```diff
@@ -1,14 +1,10 @@
-import { redirect, type ActionFunctionArgs } from '@remix-run/node'
+import { type ActionFunctionArgs } from '@remix-run/node'
 import { authenticator } from '#app/utils/auth.server.ts'
 import { handleMockAction } from '#app/utils/connections.server.ts'
 import { ProviderNameSchema } from '#app/utils/connections.tsx'
 import { getReferrerRoute } from '#app/utils/misc.tsx'
 import { getRedirectCookieHeader } from '#app/utils/redirect-cookie.server.ts'
 
-export async function loader() {
-	return redirect('/login')
-}
-
```

Comment:
Why remove this?

Reply:
Playwright failed when the login redirect was present, so I had to remove it.